.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
Don't you think it is already suitable to use venv? So adding .venv to the .gitignore? Or perhaps, this should be added to the user-specific .gitignore because it's pretty standard? Do you know what's the best practise?